alacritty_terminal/src/term/mod.rs

```diff
@@ -1571,6 +1571,11 @@ impl<T: EventListener> Handler for Term<T> {
         let old_col = self.grid.cursor.point.column.0;
         for _ in 0..count {
             let mut col = self.grid.cursor.point.column;
+
+            if col == 0 {
+                break;
+            }
```

Comment:
Did you fix a bug in our existing parser here? What impact did this have? Might need a changelog entry too?

Reply:
it's a perf optimization to break once you've reached the start. It's not a bug.